docs/getting-started-step-by-step/00-overview.md

```diff
@@ -0,0 +1,44 @@
+# Overview Getting Started: Step-by-Step
+
+The following guide provides an easy, step-by-step path for developers and platform engineers to get familiar with Crossplane. It complements the current Getting Started guide, which focuses on getting an RDS Instance up and running quickly. In this example will use Object Storage as a simple example with quick turn around times.
+
+In this guide we are going to all important aspects of Crossplane: Managed Resources, Provider, ProviderConfigs, Compositions, CompositeResourceDefinitions (XRD), CompositeResources (XR), Claims and Configurations. 
+
+Here is the full architectural overview: 
+
+![architecture-overview](00-architecture-overview.jpg)
+
+Don't worry about the different terms. We will introduce them on our way.
+
+We will use this condensed form throughout the guide: 
+
+![architecture-overview-condensed](architecture-all.jpg)
+
+## Parts & Steps
+
+The guide consists of 4 parts with individual steps:
+
+* 1.) [Setup](01-setup.md): Setting up Crossplane on Kubernetes.
+    * Access a Kubernetes cluster
+    * Install the Crossplane CLI
+    * Install the Crossplane Runtime
```

Comment:
I'd prefer to avoid calling Crossplane (i.e. the controllers) as "the Crossplane runtime" to avoid it being conflated with https://github.com/crossplane/crossplane-runtime, which are our core Crossplane libraries.

Reply:
Do we have something more specific than Crossplane? WDYT about "the Crossplane system"?
I would like to explicitly refer to the namespace we create and the things we install initially. I also want to refer to these later on. e.g. Make sure you have a fresh Crossplane Runtime / System running.

Reply:
We don't - is this common? I typically don't see folks talk about installing (for example) "the Istio runtime" or "the KubeVela system". As best I can tell it's common to simply refer to the set of controllers/deployments by the project name - e.g. "Crossplane" and then use a modifier like "the Crossplane CLI" for additional components.

"The Crossplane controller" or "Crossplane deployment" may make sense if we need a modifier?